src/Symfony/Component/Process/Process.php

```diff
@@ -1220,6 +1223,36 @@ public function getStartTime(): float
         return $this->starttime;
     }
 
+    /**
+     * setWinOptions sets windows proc_open specific options
+     * useful with the option create_new_console to open a process and an detach it
+     * from your php script so it can run in background
+     * requires minimum PHP 7.4.4.
+     */
+    public function setWinOptions(array $options)
+    {
+        if ('\\' !== \DIRECTORY_SEPARATOR) {
+            throw new LogicException('setWinOptions is only for setting Windows proc_open options.');
+        }
+
+        $defaultOptions = $this->options;
+        $existingWinOptions = ['suppress_errors', 'bypass_shell', 'blocking_pipes', 'create_process_group', 'create_new_console'];
+
+        foreach ($options as $key => $value) {
+            if (!\in_array($key, $existingWinOptions)) {
+                $this->options = $defaultOptions;
+                throw new LogicException($key.' is not a valid windows option for proc_open.');
```

Comment:
`sprintf('Invalid option "%s" passed to "%s()". Supported options are "%s".', $key, __METHOD__, implode('", "', $existingOptions));`

Reply:
I've updated this to your recommendation.